integration-msk-to-lambda/main.rs

```diff
@@ -0,0 +1,56 @@
+use aws_lambda_events::event::kafka::KafkaEvent;
+use lambda_runtime::{run, service_fn, tracing, Error, LambdaEvent};
+use base64::prelude::*;
+use serde_json::{Value};
+use tracing::{info};
+
+/// Pre-Requisites:
+/// 1. Install Cargo Lambda - see https://www.cargo-lambda.info/guide/getting-started.html
+/// 2. Add packages tracing, tracing-subscriber, serde_json, base64
+///
+/// This is the main body for the function.
+/// Write your code inside it.
+/// There are some code example in the following URLs:
+/// - https://github.com/awslabs/aws-lambda-rust-runtime/tree/main/examples
+/// - https://github.com/aws-samples/serverless-rust-demo/
+
+async fn function_handler(event: LambdaEvent<KafkaEvent>) -> Result<Value, Error> {
+
+    let payload = event.payload.records;
+
+    for (_name, records) in payload.iter() {
+
+        for record in records {
+
+         let record_text = record.value.as_ref().unwrap();
+         info!("Record: {}", &record_text);
+
+         // perform Base64 decoding
+         let record_bytes = BASE64_STANDARD.decode(record_text).unwrap();
+         let message = std::str::from_utf8(&record_bytes).unwrap();
+         info!("Message: {}", message);
+        }
+
+    }
+
+    Ok(().into())
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+
+    tracing_subscriber::fmt().json()
```

Comment:
Use 'tracing::init_default_subscriber();' to replace this. This is a new function provided by Rust Runtime to simplify this part.